replace-tokens/replace-tokens.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: replace-tokens
+spec:
+  workspaces:
+    - name: source
+    - name: token
+  params:
+    - name: inputFilePath
+      type: string
+      description: The file path whose value needs to be replaced
+  steps:
+    - name: replace-tokens
+      image: quay.io/vinamra2807/replace-tokens:latest
```

Comment:
I think we need to come up with some guidance about tagging images in Tasks. By using :latest (or any tag) users are opening themselves up to breakages if the image owner pushes an update. 

It won't block this PR but we should recommend better assurances here for future Task authors.

Reply:
@ImJasonH could you please guide me what tag shall I use for the image?

Reply:
For now :latest is fine. My comment was just expressing that at some point we should have guidance that these images be expressed by immutable digest (`@sha256:...`) rather than mutable tag.

You don't have to do that now though.

Reply:
Okay